packages/workspace/src/browser/workspace-service.ts

```diff
@@ -401,3 +467,73 @@ export interface WorkspaceInput {
     preserveWindow?: boolean;
 
 }
+
+interface WorkspaceData {
+    folders: Array<{ path: string }>;
+    // TODO add workspace settings settings?: { [id: string]: any };
+}
+
+namespace WorkspaceData {
+    const WorkspaceDataSchema = {
```

Comment:
`WorkspaceData.WorkspaceDataSchema` -> `WorkspaceData.Schema` - repeating `workspaceData` in each variable name of `WorkspaceData` is redundant